homeassistant/components/hassio/config_flow.py

```diff
@@ -0,0 +1,22 @@
+"""Config flow for Home Assistant Supervisor integration."""
+import logging
+
+from homeassistant import config_entries
+
+from . import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Home Assistant Supervisor."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+
+    async def async_step_hassio(self, user_input=None):
```

Comment:
Not sure there, maybe @MartinHjelmare and @emontnemery can help there. The `hassio` step would not be the trigger as we do not send a discovery from the supervisor and we should not mix this source. I think there needs a new step `async_step_system` which get trigger by `async_setup`?

Reply:
We don't have a system source and I don't know that it makes sense to add as part of this PR. I am using a hardcoded string for now so that the config flow is less misleading, this can be switched to use a constant if we decide to add one